src/encoded/static/components/search.js

```diff
@@ -605,6 +610,17 @@ const DatasetComponent = (props, reactContext) => {
                 if (dataset.assay_term_name) {
                     assays.push(dataset.assay_term_name);
                 }
+                if (dataset.crispr_screen_readout) {
+                    crisprReadout.push(dataset.crispr_screen_readout);
+                }
+                if (dataset.examined_loci && dataset.examined_loci.length > 0) {
+                    dataset.examined_loci.forEach((loci) => {
+                        examineLociGene.push(loci.gene.symbol);
+                    });
```

Comment:
Can you do: `let examineLociGene = dataset.examined_loci.map((loci) => loci.gene.symbol);` here?
I haven't tried this, I'm not sure if it would work depending on what the data looks like, but it could be shorter if it does.

Reply:
Oh I tried it but it only got the first examined_loci gene from each dataset. I think we would want all examine_loci gene (if more than one) from each dataset in the series.